contents/docs/web-analytics/api.mdx

```diff
@@ -0,0 +1,413 @@
+---
+title: Web Analytics API
```

Comment:
Is this the right name / place for this? I wonder how people will come looking for this info. Is it...

"I want to use PostHog's web analytics and serve that to my customers, where is their web analytics API?"

or is it...

"I want to serve data in PostHog to my customers. How do I get that data? -> Oh, there are these fast/cheap queries that get me pageview and referrer data, cool, I will start with that, and add custom insights later using their /query endpoint."

I think (though open to disagreements) that it might be more of the latter.

Reply:
Y'all have the data, but is "host" enough of a filter to act as user-facing analytics?

If so, the best thing to do is two pages:

1. `/docs/web-analytics/embedded-analytics`: I chose embedded analytics because it is the one with the highest search volume, but client-facing analytics is also good. Focus this current doc more on embedded analytics use case, rather than being a complete API reference. Mention query API as the last option 
2. `/docs/api/web-analytics`: We already have a page, but we can add more details to help it be useful (a lot of the "Web API specific" stuff here) and move the page itself into the "popular" section.

Reply:
I think web analytics is a lot better place. to put it than basically anywhere else.